assets/queries/k8s/root_container_not_mounted_as_read_only/metadata.json

```diff
@@ -1,9 +1,9 @@
 {
   "id": "a9c2f49d-0671-4fc9-9ece-f4e261e128d0",
-  "queryName": "Root Container Not Mounted As Read-only",
+  "queryName": "Root Container Not Mounted Read-only",
```

Comment:
Can you also update the queryName in the `positive_expected_result.json`, please?